packages/listbox/src/ListboxMixin.js

```diff
@@ -252,6 +252,8 @@ const ListboxMixinImplementation = superclass =>
       super.firstUpdated(changedProperties);
 
       this.__moveOptionsToListboxNode();
+      /** @type {any[]} */
+      this._initialModelValue = this.modelValue;
```

Comment:
I am wondering if we should better use
```js
this.registrationComplete.then(() => {
  this._initialModelValue = this.modelValue;
});
```

to be more "safe" if some child maybe takes slightly longer 😅